google/ggrc-core
# GGRC-7264 Object's page is refreshed when create a person via modal window

# Issue description

Current page redirects to Person page after creating people via autocomplete-results `+ Create` button on opened edit/create modal of other object.

# Steps to test the changes

1. Open modal for creation of Program
2. Click on '+ create' for any role field
3. Fill fields in 'New Person' modal
4. Click 'Save & close'
5. Expected: person is created and set for the role field

# Solution description

Added condition for preventing redirection after Person created via `+ Create` button

# Sanity checklist




# PR Review checklist
